app/forms/registration.rb

```diff
@@ -55,16 +55,15 @@ def submit params
     params[:activity_ids] ||= {}
     params[:plans] ||= {}
     params[:registration] ||= {}
-    if admin?
-      params.permit!
-    else
-      params.except(:comment, :minor_agreement_received).permit!
```

Comment:
This line wasn't actually working, in a way that was definitely easy to miss: `comment` and `minor_agreement_received` were both keys on `:registration`, not on `params` itself, so they weren't getting excepted.